src/modules/85-sto/components/ExternalTickets/Settings/ExternalTicketSettings.tsx

```diff
@@ -99,13 +99,26 @@ const ExternalTicketSettings: React.FC<{ debounceDelay?: number }> = ({ debounce
       <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'right' }} margin={{ bottom: 'large' }}>
         <Text className={css.minWidth}>{getString('common.tickets.defaultProjectName')}</Text>
         <div className={cx(stepCss.formGroup, stepCss.lg)}>
-          <TextInput
+          <DropDown
+            items={projectItems}
             value={ticketSettings?.projectKey}
-            title="defaultProjectName"
-            name="defaultProjectName"
-            placeholder={getString('common.tickets.selectProjectName')}
-            onChange={(ev: React.ChangeEvent<HTMLInputElement>) => {
-              updateTicketSettings({ projectKey: ev.target.value })
+            disabled={isLoadingSettings || isLoadingProjects}
+            onChange={item => {
+              updateTicketSettings({ projectKey: item.value as string })
+            }}
+          />
+        </div>
+      </Layout.Horizontal>
+
+      <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'right' }} margin={{ bottom: 'large' }}>
+        <Text className={css.minWidth}>{getString('common.tickets.defaultIssueType')}</Text>
+        <div className={cx(stepCss.formGroup, stepCss.lg)}>
+          <IssueTypesDropDown
+            jiraProjectId={ticketSettings?.projectKey}
+            value={ticketSettings?.issueType}
+            disabled={isLoadingSettings || isLoadingProjects}
+            onChange={item => {
+              updateTicketSettings({ issueType: item.value as string })
             }}
           />
```

Comment:
```suggestion
        ticketSettings?.projectKey ? (
          <IssueTypesDropDown
            jiraProjectId={ticketSettings?.projectKey}
            value={ticketSettings?.issueType}
            disabled={isLoadingSettings || isLoadingProjects}
            onChange={item => {
              updateTicketSettings({ issueType: item.value as string })
            }}
          />
          ) : (
            <DropDown items={[]} />
          )
```
will avoid making invalid API calls to the metadata endpoint when no project is selected. You can then make the `jiraProjectId` prop on `IssueTypesDropDown` to be required and remove the defaulting to `''`